lib/cdo/graphics/certificate_image.rb

```diff
@@ -114,14 +145,17 @@ def create_course_certificate_image(name, course=nil, sponsor=nil, course_title=
   path = pegasus_dir('sites.v3', 'code.org', 'public', 'images', template_file)
   if prefilled_title_course?(course)
     # only need to fill in student name
-    vertical_offset = course == '20-hour' ? -115 : -110
+    vertical_offset = course == '20-hour' ? -125 : -120
```

Comment:
why did this change?

Reply:
Slight improvement to the centering of it on the certificate.